third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -631,6 +673,32 @@ func resourceComposerEnvironmentUpdate(d *schema.ResourceData, meta interface{})
 			}
 			d.SetPartial("config")
 		}
+
+    <% unless version == "ga" -%>
```

Comment:
nit: spaces -> tabs in this section as well.

Reply:
Sure. I did not notice but let my IDE fix that.